client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -30,22 +30,34 @@ export default function Dashboard(): JSX.Element {
   if (loggedInUser === undefined) return <CircularProgress />;
 
   return (
-    <Grid container item xs={12} sm={12} md={12} component="main" className={`${classes.root} ${classes.dashboard}`}>
+    <Grid
+      container
+      item
+      xs={12}
+      sm={12}
+      md={12}
+      direction="column"
+      justify="flex-start"
+      component="main"
+      className={`${classes.root}`}
+    >
       <CssBaseline />
-      <Grid item className={classes.drawerWrapper}>
+      <Grid className={classes.drawerWrapper}>
```

Comment:
Why did you remove the `item` here?

Reply:
I remove the item there due to some styling issues that were conflicting within item's declared styling.